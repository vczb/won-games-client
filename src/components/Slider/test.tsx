import { render, screen } from '@testing-library/react'

import Slider from '.'

describe('<Slider />', () => {
  it('should render the reading', () => {
    const { container } = render(<Slider />)

    expect(screen.getByRole('heading', { name: /Slider/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })

})
