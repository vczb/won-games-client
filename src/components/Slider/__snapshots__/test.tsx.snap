// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Slider /> should render the reading 1`] = `
.c0 .slick-slider {
  position: relative;
  display: block;
  box-sizing: border-box;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  -webkit-touch-callout: none;
  -khtml-user-select: none;
  -ms-touch-action: pan-y;
  touch-action: pan-y;
  -webkit-tap-highlight-color: transparent;
}

.c0 .slick-list {
  position: relative;
  display: block;
  overflow: hidden;
  margin: 0;
  padding: 0;
}

.c0 .slick-list:focus {
  outline: none;
}

.c0 .slick-list.dragging {
  cursor: pointer;
  cursor: hand;
}

.c0 .slick-slider .slick-track,
.c0 .slick-slider .slick-list {
  -webkit-transform: translate3d(0,0,0);
  -moz-transform: translate3d(0,0,0);
  -ms-transform: translate3d(0,0,0);
  -o-transform: translate3d(0,0,0);
  -webkit-transform: translate3d(0,0,0);
  -ms-transform: translate3d(0,0,0);
  transform: translate3d(0,0,0);
}

.c0 .slick-track {
  position: relative;
  top: 0;
  left: 0;
  display: block;
}

.c0 .slick-track:before,
.c0 .slick-track:after {
  display: table;
  content: '';
}

.c0 .slick-track:after {
  clear: both;
}

.c0 .slick-loading .slick-track {
  visibility: hidden;
}

.c0 .slick-slide {
  display: none;
  float: left;
  height: 100%;
  min-height: 1px;
}

.c0 [dir='rtl'] .slick-slide {
  float: right;
}

.c0 .slick-slide img {
  display: block;
}

.c0 .slick-slide.slick-loading img {
  display: none;
}

.c0 .slick-slide.dragging img {
  pointer-events: none;
}

.c0 .slick-initialized .slick-slide {
  display: block;
}

.c0 .slick-loading .slick-slide {
  visibility: hidden;
}

.c0 .slick-vertical .slick-slide {
  display: block;
  height: auto;
  border: 1px solid transparent;
}

.c0 .slick-arrow.slick-hidden {
  display: none;
}

<section
  class="c0"
>
  <h1>
    Slider
  </h1>
</section>
`;
